ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/remote/AwsConnectorExecutor.java

```diff
@@ -69,15 +70,19 @@ public AwsConnectorExecutor(Connector connector) {
         this.httpClient = MLHttpClientFactory.getAsyncHttpClient(connectionTimeout, readTimeout, maxConnection);
     }
 
+    @Override
+    public Logger getLogger() {
```

Comment:
Why do we need to expose this log?

Reply:
Logger is a required parameter to init a RetryableAction, where it's used to log debug info https://github.com/opensearch-project/OpenSearch/blob/27003270adedebd43eedbd0186c0825f2c0e8613/server/src/main/java/org/opensearch/action/support/RetryableAction.java#L192

Reply:
And we use the exposed logger at RemoteConnectorExecutor, as we init he RetryableAction here